backend/utils/seed.ts

```diff
@@ -0,0 +1,148 @@
+import prisma from './prismaClient';
+
+const db = prisma
+
+const userData = Array.from({ length: 20 }).map(() => ({
+    firstName: faker.name.firstName(),
+    lastName: faker.name.lastName(),
+    email: faker.internet.email()
+}))
+
+
+const main = async () => {
+
+    //create many users
+    await db.user.createMany({
+        data: userData
+    })
+
+    //create user: hyun, john, sarah
+    const hyun = await db.user.create({
+        data: {
+            email: 'test@gmail.com',
+            firstName: 'hyun',
+            lastName: 'test'
+        }
+    })
+
+    const john = await db.user.create({
+        data: {
+            email: 'john@gmail.com',
+            firstName: 'john',
+            lastName: 'test'
+        }
+    })
+
+    const sarah = await db.user.create({
+        data: {
+            email: 'sarah@gmail.com',
+            firstName: 'sarah',
+            lastName: 'test'
+        }
+    })
+
+    //create contact 'oliver'
+    const oliver = await db.contact.create({
+        data: {
+            firstName: 'Oliver',
+            address: 'abc 123 SF, Cali',
+            source: 'phone contact',
+            ownerID: hyun.UserID
+        }
+    })
+
+    //create many contacts for hyun
+    const contactData = Array.from({ length: 50 }).map(() => ({
+        firstName: faker.name.firstName(),
+        lastName: faker.name.lastName(),
+        email: faker.internet.email(),
+        address: faker.address.streetAddress(),
+        ownerID: hyun.UserID,
+        source: 'phone contact'
+    }))
+
+    await db.contact.createMany({
+        data: [...contactData]
+    })
+
+    //create many contacts for sarah
+    await db.contact.createMany({
+        data: contactData.map(obj => 
+            ({ ...obj, ownerID: sarah.UserID })
+        )
+    })
+
+    //create birthday plan for contact oliver
+    await db.plan.create({
+        data: {
+            description: 'Birthday plan for pedro!',
+            startDate: new Date('03-07-2023'),
+            endDate: new Date('03-20-2023'),
+            specialEventType: 'BIRTHDAY',
+            contactID: oliver.ContactID,
+        }
+    })
+
+    //create birthday plan for user john
+    const johnPlan = await db.plan.create({
+        data: {
+            description: "John's graduation",
+            startDate: new Date('03-07-2023'),
+            endDate: new Date('03-20-2023'),
+            specialEventType: 'BIRTHDAY',
+        }
+    })
+
+    //hyun becomes a pledge for john's plan
+    await db.pledge.create({
+        data: {
+            planID: johnPlan.PlanID,
+            userID: hyun.UserID,
+            inviteStatus: 'ACCEPTED',
+            role: 'PLAN_LEADER'
+        }
+    })
+
+    //hyun sends john as a friend first
+    const hyunAddsJohn = await db.userRelationship.create({
+        data: {
+            userID: hyun.UserID,
+            friendsWithID: john.UserID,
+            relationshipStatus: "PENDING_REQUEST"
```

Comment:
let's avoid these magic strings! you already defined an enum, so let's use that here

Reply:
The prisma client display the enum list when I enter an empty string. 
![image](https://user-images.githubusercontent.com/106122976/224221283-d0a38189-cef7-41be-8d74-c4d35d98a8d7.png)

Do you want me to use the typescript enum here

Reply:
yeah using an enum value in general would be preferred to just having these strings float around for just a bunch of reasons

Reply:
Imported prisma enum 
![image](https://user-images.githubusercontent.com/106122976/224221902-52c28dc7-c642-469d-a7e2-4a06eab2041a.png)

![image](https://user-images.githubusercontent.com/106122976/224221888-bd018031-0574-4161-9e09-6c8b8bf4582b.png)

Reply:
sick!